sql/moz-fx-data-shared-prod/adjust_derived/adjust_derived_v1/query.py

```diff
@@ -0,0 +1,305 @@
+"""Adjust data - download deliverables, clean and upload to BigQuery."""
+import csv
+import json
+import os
+import tempfile
+from argparse import ArgumentParser
+
+import requests
+from google.cloud import bigquery
+
+API_URI = "api.adjust.com"
+ENDPOINT = "kpis"
+API_VERSION = "v1"
+
+"""This is a list of dictionaries.
+Keys are the app_name, and app_token in the form:
+{"app_name":"<specific app name from Adjust dash>". "app_token":"<unique Adjust app token>}"
+Mozilla has 16 apps on Adjust.
+"""
+
+APP_TOKEN_LIST = os.environ.get("ADJUST_APP_TOKEN_LIST")
+
+"""This is Marlene's personal Adjust API Token. We don't have a special service account API Token."""
+API_TOKEN = os.environ.get("ADJUST_API_TOKEN")
+
+CSV_FIELDS = []
+CSV_FIELDS = [
+    "date",
+    "app",
+    "app_token",
+    "network",
+    "network_token",
+    "campaign",
+    "campaign_token",
+    "adgroup",
+    "adgroup_token",
+    "creative",
+    "creative_token",
+    "country",
+    "os",
+    "device",
+    "clicks",
+    "installs",
+    "limit_ad_tracking_install_rate",
+    "click_conversion_rate",
+    "impression_conversion_rate",
+    "sessions",
+    "daus",
+    "waus",
+    "maus",
+]
+
+
+def read_json(filename: str) -> dict:
+    """Read JSON file."""
+    with open(filename, "r") as f:
+        data = json.loads(f.read())
+    return data
+
+
+def write_dict_to_csv(json_data, filename):
```

Comment:
I noticed in `read_json()` definition you're using type hinting. Since this is new code it'd be great if we could use it for all function definitions in this file :)

Reply:
It would be great to do that, but... the write_dict_to_csv doesn't return anything, so how would type hinting work here?

Reply:
You can always say that the function return value is `None`:
```
def my_fun(arg: str) -> None:
   [function_body]
```

Reply:
And I think it's nice to include as it indicates that it does not return anything.